saleor/plugins/models.py

```diff
@@ -1,7 +1,7 @@
 from django.contrib.postgres.fields import JSONField
 from django.db import models
 
-from saleor.core.permissions import ExtensionsPermissions
+from saleor.core.permissions import PluginsPermissions
```

Comment:
Can you change improts from this file to relative?

Reply:
Yup, we use relative imports across the project, except for tests.